js/index.js

```diff
@@ -1,42 +1,134 @@
-    "vision-h4":"Vision",
-    "vision-content": "Vision content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
+  'main-content': {
+    'features-h4': 'Features',
+    'features-content':
+			'Features content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.',
+    'about-h4': 'About',
+    'about-content':
+			'About content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.',
+    'middle-img-src': 'img/mid-page-accent.jpg',
+    'services-h4': 'Services',
+    'services-content':
+			'Services content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.',
+    'product-h4': 'Product',
+    'product-content':
+			'Product content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.',
+    'vision-h4': 'Vision',
+    'vision-content':
+			'Vision content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.',
   },
-  "contact": {
-    "contact-h4" : "Contact",
-    "address" : "123 Way 456 Street Somewhere, USA",
-    "phone" : "1 (888) 888-8888",
-    "email" : "sales@greatidea.io",
+  contact: {
+    'contact-h4': 'Contact',
+    address: '123 Way 456 Street Somewhere, USA',
+    phone: '1 (888) 888-8888',
+    email: 'sales@greatidea.io',
   },
-  "footer": {
-    "copyright" : "Copyright Great Idea! 2018"
+  footer: {
+    copyright: 'Copyright Great Idea! 2018',
   },
 };
 
+// Nav Bar
 // Example: Update the img src for the logo
-let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+const logo = document.getElementById('logo-img');
+logo.setAttribute('src', siteContent.nav['img-src']);
+const navBar = document.querySelector('nav');
+const navLinks = navBar.querySelectorAll('a');
+const navObj = siteContent.nav;
+const navValues = Object.values(navObj);
+[...navLinks].forEach((link, index) => {
+  link.textContent = navValues[index];
+  link.style.color = 'green'; // change link font colour to green.
+});
+
+// Add 2 elements to the nav bar
+const BestLink = document.createElement('a');
```

Comment:
Avoid using capitalised letters for variables, these should be reserved for constructor functions and classes i.e. things invoked using ```new```